src/styles/index.styl

```diff
@@ -35,7 +36,7 @@
 }
 
 .coz-bar-hidden
-  @extend $hidden
+  @extend .u-hidden
 
 .coz-btn-regular
   @extend $button
```

Comment:
You could also extend `.c-btn` instead of $button since you're using 4.0.0

Reply:
👍

Reply:
Even better, these styles seem to be unused, so I removed them.